loadtest/control/simulcontroller/scheduled_posts.go

```diff
@@ -0,0 +1,184 @@
+package simulcontroller
+
+import (
+	"fmt"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/user"
+	"github.com/mattermost/mattermost/server/public/model"
+	"math/rand"
+)
+
+func (c *SimulController) createScheduledPost(u user.User) control.UserActionResponse {
+	if ok, resp := control.ScheduledPostsEnabled(u); resp.Err != nil {
+		fmt.Println("createScheduledPost: ScheduledPostsEnabled error", resp.Err)
+		return resp
+	} else if !ok {
+		fmt.Println("createScheduledPost: ScheduledPosts not enabled")
+		return control.UserActionResponse{Info: "scheduled posts not enabled"}
+	}
+
+	channel, err := u.Store().CurrentChannel()
+	if err != nil {
+		fmt.Println("createScheduledPost: CurrentChannel error", err)
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	fmt.Println("createScheduledPost: got the channel", channel.Id)
+	var rootId = ""
+	if rand.Float64() < 0.25 {
+		post, err := u.Store().RandomPostForChannel(channel.Id)
+		if err == nil {
+			if post.RootId != "" {
+				rootId = post.RootId
+			} else {
+				rootId = post.Id
+			}
+		}
+	}
+
+	if err := sendTypingEventIfEnabled(u, channel.Id); err != nil {
+		fmt.Println("createScheduledPost: sendTypingEventIfEnabled error", err)
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	message, err := createMessage(u, channel, false)
+	if err != nil {
+		fmt.Println("createScheduledPost: createMessage error", err)
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	scheduledPost := &model.ScheduledPost{
+		Draft: model.Draft{
+			Message:   message,
+			ChannelId: channel.Id,
+			RootId:    rootId,
+			CreateAt:  model.GetMillis(),
+		},
+		ScheduledAt: loadtest.RandomFutureTime(17_28_00_000, 10),
```

Comment:
What does `17_28_00_000` mean?

Reply:
Its 2 days in milliseconds. I've created a const for this to make it more readable.